### ISSUE TITLE
All page content should be contained by landmarks

### ISSUE THREAD

**Issue by christinach** [pulibrary/static-tables#104]
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Ensures the document has a main landmark

### Severity
High

### WCAG Success Criterion
[SC 2.4.1 Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
When this Success Criterion is not satisfied, it may be difficult for people with some disabilities to reach the main content of a Web page quickly and easily:

Screen reader users who visit several pages on the same site can avoid having to hear all heading graphics and dozens of navigation links on every page before the main content is spoken.
People who use only the keyboard or a keyboard interface can reach content with fewer keystrokes. Otherwise, they might have to make dozens of keystrokes before reaching a link in the main content area. This can take a long time and may cause severe physical pain for some users.
People who use screen magnifiers do not have to search through the same headings or other blocks of information to find where the content begins each time they enter a new page.
People with cognitive limitations as well as people who use screen readers may benefit when links are grouped into lists

### Recommended fix
Add page content in landmarks

Element location: .header
<div class="header navbar navbar-dark bg-black">

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)
